docs/generators/typescript-axios.md

```diff
@@ -20,6 +20,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 | ------ | ----------- | ------ | ------- |
 |allowUnicodeIdentifiers|boolean, toggles whether unicode identifiers are allowed in names or not, default is false| |false|
 |apiPackage|package for generated api classes| |null|
+|axiosVersion|Use this property to override the axios version in package.json| |^1.6.1|
```

Comment:
Non-blocking suggestion - a bit more detail on acceptable values could be helpful here, something like:

> Value may be [any valid NPM semver string](https://github.com/npm/node-semver?tab=readme-ov-file#ranges) that matches an [existing version of `axios`](https://www.npmjs.com/package/axios?activeTab=versions).